.gitignore

```diff
@@ -177,13 +150,12 @@ pip-selfcheck.json
 # .nfs files are created when an open file is removed but is still being accessed
 .nfs*
 
-.*
```

Comment:
Had to remove this one because it was conflicting with adding the `.pylintrc` to VC, and it will undoubtedly cause more problems in the future (like adding a `.circleci` dir)